.ci/setup_env.sh

```diff
@@ -119,12 +74,21 @@ fi
 # Install Test::Nginx
 # -------------------
 if [[ "$TEST_SUITE" == "pdk" ]]; then
-  echo "Installing CPAN dependencies..."
+  CPAN_DOWNLOAD=$DOWNLOAD_ROOT/cpanm
+  mkdir -p $CPAN_DOWNLOAD
+  wget -O $CPAN_DOWNLOAD/cpanm https://cpanmin.us
   chmod +x $CPAN_DOWNLOAD/cpanm
+  export PATH=$CPAN_DOWNLOAD:$PATH
+
+  echo "Installing CPAN dependencies..."
   cpanm --notest Test::Nginx &> build.log || (cat build.log && exit 1)
   cpanm --notest --local-lib=$TRAVIS_BUILD_DIR/perl5 local::lib && eval $(perl -I $TRAVIS_BUILD_DIR/perl5/lib/perl5/ -Mlocal::lib)
 fi
 
+if [ ! "$(ls -A $KONG_DOWNLOAD)" ]; then
```

Comment:
Seems that `$KONG_DOWNLOAD` is not inside the download cache, therefore this check will probably not work as intended.

Reply:
it is defined as `KONG_DOWNLOAD=$DOWNLOAD_CACHE/kong` in line 7

Reply:
I don't know why this moved further down here, I think it was to maintain the part that was copy-pasted from Kong/kong's `setup_env.sh` identical. Probably better to move it back up to the `Download` section of the script (where it originally was).